Blazor/README.md

```diff
@@ -19,20 +19,20 @@ Issues can be reported on the [Github repository](https://github.com/wet-boew/cd
 
 To use, in your project: 
 
-  - Add the GoC.WebTemplate.Blazor nuget package to your Blazor project. This should automatically pull in the GoC.WebTEmplate.Entities package as well.
+  - Add the GoC.WebTemplate.Blazor nuget package to your Blazor project. This should automatically pull in the GoC.WebTEmplate.Entities dll as well.
 
   - In index.html, add the stylesheet element to the HEAD section corresponding to the desired template/environment:
 
     | Version                   | Theme                          | Environment | CSS URL                                                                                                                  |
     | ------------------------- | ------------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------ |
-    | specific<br>(e.g. v4_1_0) | gcweb                          | prod        | `<link rel="stylesheet" href="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v5_0_0/cdts/cdts-app-styles.css">`     |
-    | specific<br>(e.g. v4_1_0) | gcweb                          | prod        | `<link rel="stylesheet" href="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v5_0_0/cdts/cdts-styles.css">`         |
-    | specific<br>(e.g. v4_1_0) | gcweb                          | esdcprod    | `<link rel="stylesheet" href="https://cdts.service.canada.ca/app/cls/WET/gcweb/v5_0_0/cdts/cdts-app-styles.css">`        |
-    | specific<br>(e.g. v4_1_0) | gcweb                          | esdcprod    | `<link rel="stylesheet" href="https://cdts.service.canada.ca/app/cls/WET/gcweb/v5_0_0/cdts/cdts-styles.css">`            |
-    | specific<br>(e.g. v4_1_0) | gcintranet                     | prod        | `<link rel="stylesheet" href="https://cdts.service.canada.ca/app/cls/WET/gcintranet/v5_0_0/cdts/cdts-styles.css">`       |
-    | specific<br>(e.g. v4_1_0) | gcintranet                     | esdcprod    | `<link rel="stylesheet" href="https://templates.service.gc.ca/app/cls/WET/gcintranet/v5_0_0/cdts/cdts-styles.css">`      |
-    | specific<br>(e.g. v4_1_0) | gcintranet<br>(ESDC sub-theme) | prod        | `<link rel="stylesheet" href="https://cdts.service.canada.ca/app/cls/WET/gcintranet/v5_0_0/cdts/cdts-esdc-styles.css">`  |
-    | specific<br>(e.g. v4_1_0) | gcintranet<br>(ESDC sub-theme) | esdcprod    | `<link rel="stylesheet" href="https://templates.service.gc.ca/app/cls/WET/gcintranet/v5_0_0/cdts/cdts-esdc-styles.css">` |
+    | specific<br>(e.g. v5_0_0) | gcweb                          | prod        | `<link rel="stylesheet" href="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v5_0_0/cdts/cdts-app-styles.css">`     |
```

Comment:
For this table, I know you don't have a "mode" in Blazor and that's why you removed the column, but there is no nothing to helps users pick the proper CSS if they want the "App" template.  Maybe that column should stay, it can just be called "type" instead?  Thoughts?

Reply:
Updated